### PR TITLE
Update for release

### PR DESCRIPTION
**Pull request by tbekolay**
This PR gets nengo_reference good to go for a release.

Despite being just around 1000 lines of Python (including tests) it manages to pass 130 / 199 tests! Nice.

@tcstewar, any objections to putting this on PyPI? And any objections to moving this to http://github.com/nengo? We can also take this opportunity to rename the project... it may be a bit misleading to call it nengo_reference.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
+===============
+nengo_reference
+===============
+
+An example backend implementation for Nengo.
+
+This takes a neural model described
+using the Nengo framework,
+uilds it into an actual neural simulation,
```
Comment (tcstewar) [inline README.rst]: uilds -> builds

**Comment by tcstewar** [comment]
> @tcstewar, any objections to putting this on PyPI? And any objections to moving this to http://github.com/nengo?

No objections at all.  :)  Thanks for cleaning it up like this!

> We can also take this opportunity to rename the project... it may be a bit misleading to call it nengo_reference.

Hmm...  not sure what else to call it.  I'd intended it as a reference implementation in the sense of it being a simple, readable implementation to point people at if they are interested in understanding what's going on behind the scenes.  It also turned out to be pretty useful as a starting point for people making their own backends.  So that's kind of like a reference implementation, but then we'd need some other name to refer to nengo/nengo.

**Comment by tbekolay** [comment]
> Hmm... not sure what else to call it

I'm not super sure either, but I do worry about the confusion with referring to them both as reference implementations. We could call this `nengo_simplified`? Or emphasize the starting point aspect as `nengo_template` or `nengo_backend_template` ... or, we could rip out the backend from `nengo/nengo` and make it `nengo_sigops` or something like that... that's a big change though.

**Comment by tcstewar** [comment]
Hmm..  I think I could see ```nengo_simplified``` working...  or perhaps ```nengo_clean``` or ```nengo_transparent```....  The thesaurus also suggests ```nengo_intelligible``` and ```nengo_lucid```, but that might be a bit much...

**Comment by tbekolay** [comment]
Let's get some more opinions, then, before moving forward. Thoughts, @hunse @jgosmann @drasmuss @xchoo @celiasmith @studywolf?

**Comment by hunse** [comment]
`nengo_bones`, because it's a bare-bones implementation. Or `nengo_whiskey`, because it's a distilled version of Nengo :smile: 

I would go with `nengo_simple` over `nengo_simplified`. Could also do `nengo_basic`. I was thinking `nengo_base` or `nengo_core`, too, but those sound like this project is the core of other Nengo backends or something. `nengo_stripped` sounds a bit too racy :smile: 

I think I like `nengo_simple` the best so far, I think.

**Comment by xchoo** [comment]
What about ```nengo-distilled``` then?

**Comment by jgosmann** [comment]
I'd vote for `nengo_simple`. Though, I like `nengo_distilled` too, the name seems to me a bit too fancy :tophat: for this project.

**Comment by hunse** [comment]
> What about nengo-distilled then?

I think it's not direct enough; a bit too ornate and jargon-y (I was trying to make that argument by hyperbole, which probably wasn't the best idea). It is catchy, though.

I think a simple repository deserves a simple name. And it seems @jgosmann can read my mind...

**Comment by celiasmith** [comment]
+1. Distilled

**Comment by studywolf** [comment]
I like distilled too!

**Comment by tbekolay** [comment]
I'm +1 distilled too; "simple" has a bit of a negative connotation, while stuff like "transparent" gives nengo/nengo a negative connotation. Distilled is positive, while not disparaging nengo/nengo. Sound good @tcstewar?

**Comment by tcstewar** [comment]
I'm liking ```nengo_distilled```.  :)

**Pull request merged**